File: OP_PROJECT_PYTHON/src/game/Animals/Sheep.py
from src.game.Animal import Animal


class Sheep(Animal):
    SHEEP_STRENGTH = 4
    SHEEP_INIT = 4
    SHEEP_TYPE = "Sheep"

    def __init__(self, position, world):
        super().__init__(Sheep.SHEEP_TYPE, Sheep.SHEEP_STRENGTH, Sheep.SHEEP_INIT, position, world)

File: OP_PROJECT_JAVA/src/game/Animals/Sheep.java
package game.Animals;

import game.Animal;
import game.Position;
import game.World;

public class Sheep extends Animal {
    private static final int SHEEP_STRENGTH = 4;
    private static final int SHEEP_INIT = 4;
    private static final String SHEEP_TYPE = "Sheep";


    public Sheep(Position position, World world) {
        super(SHEEP_TYPE, SHEEP_STRENGTH, SHEEP_INIT, position, world);
    }
}
